whitepapers/Tables.md

```diff
@@ -1,5 +1,114 @@
 # SQL Tables
 
+## Template
 
+### {ModelName}
+{Description}
 
+| Column | Type | Indexed | Nullable | FK | Default | Description |
+|:-------|:----:|:-------:|:--------:|:--:|:-------:|:------------|
+| | | ✓ | ✓ | ✓ | | |
 
+#### Other indices
+* `{column_name}`, `{column_name}`, ... [(unique)]
+* ...
+
+## Data
+
+### SourceUniqueIdentifier (SUID)
+Identifier for a specific document from a specific source.
+
+| Column | Type | Indexed | Nullable | FK | Default | Description |
+|:-------|:----:|:-------:|:--------:|:--:|:-------:|:------------|
+| `source_doc_id` | text |  |  |  |  | Identifier given to the document by the source |
+| `ingest_config_id` | int |  |  | ✓ |  | IngestConfig used to ingest the document |
```

Comment:
That's a very good point but I would want to fix that at the consolidator level. This revolves more around the issue of how to tell partial data vs subtractive data.

I think it really comes down to detecting omission of fields vs empty fields.

This may be a bit of a hack. Let's say we have the following for an agent:

```javascript
{
    "@id": "_:122",
    "@type": "agent",
    "name": "Bill",
    "affiliations": []
}
```

vs

```javascript
{
    "@id": "_:122",
    "@type": "agent",
    "name": "Bill",
}
```

The first is explicitly stating there is a lack of affiliations while the second just doesn't have affiliation data.